### PR TITLE
Ported eraUt1tai

### LINKED ISSUE #1: Excise ERFA
Replace the calls to ERFA by pure Julia functions:

* [ ] `eraD2dtf`
* [X] ~`eraDat`~ #31 
* [X] ~`eraDtdb`~ #14 
* [X] ~`eraDtf2d`~ #36
* [X] ~`eraTaitt`~ #4 
* [X] ~`eraTaiut1`~ #8 
* [X] ~`eraTaiutc`~ ~#13~ #39
* [X] ~`eraTcbtdb`~ #25
* [X] ~`eraTcgtt`~ #9 
* [X] ~`eraTdbtcb`~ #24
* [X] ~`eraTdbtt`~ #14 
* [X] ~`eraTttai`~ #6 
* [X] ~`eraTttcg`~ #10 
* [X] ~`eraTttdb`~ #20
* [X] ~`eraTtut1`~ #21 
* [X] ~`eraUt1tai`~ #7 
* [X] ~`eraUt1tt`~ #22 
* [x] ~`eraUt1utc`~ #40
* [X] ~`eraUtctai`~ ~#15~ #37
* [X] ~`eraUtcut1`~ #38

### PR DESCRIPTION
**Pull request by prakharcode**
Extends #4 
Progress towards #1 (EDIT: If you write "fixes" the issue would be closed automatically. Cheers, Helge) 

Added `ut1tai` in Julia and its test.

### CODE REVIEW

**Comment by helgee** [comment]
I think you will need to rebase your branch against master. Your other commits should not be in this PR.

File: src/conversions.jl [hunk 1]
```diff
@@ -7,8 +7,16 @@ using ..LeapSeconds
 
 
 """
+<<<<<<< HEAD
```
Comment (giordano) [inline src/conversions.jl]: There is still this merge conflict to fix.

I think you're discovering why it's a very bad idea to work on `master` branch :smile:

Comment (prakharcode) [reply src/conversions.jl]: Yes! I'm regretting it now! I'll fix this asap! And then, after getting a merge I'll clone the master branch again and then start working with new functions(not master this time at all), sounds good?

Comment (giordano) [reply src/conversions.jl]: You should use `master` only to keep in sync with upstream `master` and start a new feature branch from the most recent `master`.  That's how `master` is usually used on GitHub.

Regarding how to fix your issue, I think that a hard reset on `master` using upstream as `origin` should be enough (`git reset --hard origin/master`, if `origin` remote is this repository).  However, if you feel more safe, just clone again this repository.  In any case, do this *after* the PR will be merged :wink:

**Comment by prakharcode** [comment]
I guess I should close this PR and start again.

File: src/conversions.jl (hunk 1 above)
Comment (prakharcode) [reply src/conversions.jl]: I tried to fix it and it got worse! :disappointed:

**Comment by prakharcode** [comment]
As soon as you review this I'll create a PR for `eraTaiut1` :smile:

**Comment by helgee** [comment]
That's quite a mess 😂 But I think I can just squash-merge this as @giordano suggested. @prakharcode you will need to clean-up your fork, though.

**Pull request merged**